### PR TITLE
Spelling: A user, -removed

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/templates/manage-access.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
   </tbody>
   </table>
   <div class="panel-body">
-  <p class="helptext">{% trans "The user will be removed from the project once all user permissions are removed." %}</p>
+  <p class="helptext">{% trans "The user will be removed from the project once all user permissions are." %}</p>
```
Comment (nijel) [inline weblate/templates/manage-access.html]: Are you sure with this one? It sounds weird to me.

Comment (comradekingu) [reply weblate/templates/manage-access.html]: I think if I have to argue that I loose.

Comment (comradekingu) [inline weblate/templates/manage-access.html]:
```suggestion
  <p class="helptext">{% trans "How about "Once all its permissions are removed, the user will be removed from the project"?." %}</p>
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**